code-challenges.test.js

```diff
@@ -1,56 +1,216 @@
+// ● fruitArray › creates a function that takes in an array of words and a single letter and returns all the words that contain that particular letter
+//   ReferenceError: fruitArray is not defined
 // b) Create the function that makes the test pass.
+//function must sort through string and return word with certain letter 
+//"a" or "e" from the fruit array list. 
+//use .filter
+//it is not passing 
+// 
+// fruits = (array, string) => {
+//     const lettersorter = array.filter(function(string) {
+//     return array
+//     }
+//   return lettersorter
+//   console.log(fruitsArray(arrayOfWords1,letterA));
+//   console.log(fruitsArray(arrayOfWords2,letterE));
+
+//2nd attempt 
+
+
+  const fruitArray = (array,string) => {
+     return array.filter(value => value.toLowerCase().includes(string))
+  }
+
+//   PASS  ./code-challenges.test.js
+//   fruitArray
+//     ✓ creates a function that takes in an array of words and a single letter and returns all the words that contain that particular letter (1 ms)
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+// Snapshots:   0 total
+// Time:        0.153 s, estimated 1 s
+// Ran all test suites.
+// ✨  Done in 0.92s.
 
 
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
 
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
-// Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
-// Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
-
-
-// b) Create the function that makes the test pass.
+// const hand1 = [5, 5, 5, 3, 3]
+// // // Expected output: true
+// const hand2 = [5, 5, 3, 3, 4]
+// // // Expected output: false
+// const hand3 = [5, 5, 5, 5, 4]
+// // // Expected output: false
+
+
+//pseude code:
+//Create describe function-"fullHouseGame"
+//Create it statement-"Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”
+//A full house is exactly one pair and one three of any kind. 
+//Add const provided 
+//Add expects provided 
+//Add expected output 
+// create a conditonal statment to check if indexes equal each other 
+// so if index 0 equals 1, and index 3, equals 4 
+// so either true or false 
+
+
+//  const hand1 = [5, 5, 5, 3, 3]
+// // // Expected output: true
+// const hand2 = [5, 5, 3, 3, 4]
+// // // Expected output: false
+// const hand3 = [5, 5, 5, 5, 4]
+// // // Expected output: false
+
+describe("fullHouseGame", () => {
+  it("Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.", () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    // Expected output: true
+    const hand2 = [5, 5, 4, 3, 3]
```

Comment:
I would like to encourage you to keep working on this problem to get this modified test variable to pass.  This test variable should provide a false output and currently your function will return true for it.